src/Microsoft.AspNetCore.Sockets/HttpConnectionDispatcher.cs

```diff
@@ -225,8 +225,11 @@ private Task ProcessGetId(HttpContext context)
                     format,
                     endOfMessage: true);
 
-                await state.Application.Output.WriteAsync(message);
-
+                // REVIEW: Do we want to return a specific status code here if the connection has ended?
+                if (await state.Application.Output.WaitToWriteAsync())
```

Comment:
I'd rather not rely on the implementation details to make the code correct. If we change to a bounded channel, this could break, so let's be defensive.